.github/workflows/README.md

```diff
@@ -34,14 +34,14 @@ Pushes to the main [pypi.org](https://pypi.org/project/micropython-europi/#histo
 ### Release process
 
 1. Decide which version number to release.
-2. Update [`software/firmware/version.py`](/software/firmware/version.py) with the new version number.
-3. Update [`changelog.md`](CHANGELOG.md) with a description of the release.
```

Comment:
Is there a reason we're removing this step form the release process? Seems unrelated to the new uf2 file

Reply:
Oh, I see the "Update the generated Github Release" as the second to last step, I guess this replaces the release notes?. I don't know what this is. Should there be a link to it? or a description on how to publish?

Reply:
Good call. I've added a link to the releases page. The rest is self-explanatory. Edit the release notes as described in the docs and the select either Publish or Save Draft to continue editing.